content/docs/operating/configuration.md

```diff
@@ -268,6 +272,48 @@ Consul setups, the relevant address is in `__meta_consul_service_address`.
 In those cases, you can use the [relabel](#target-relabeling-relabel_config)
 feature to replace the special `__address__` label.
 
+### Kubernetes SD configurations `<kubernetes_sd_config>`
+
+CAUTION: Kubernetes SD is in beta: breaking changes to configuration are possible
```

Comment:
How about s/possible/still likely/. Breaking changes are still possible anywhere in Prometheus until 1.0, but less likely.